lms/djangoapps/courseware/views.py

```diff
@@ -862,6 +878,56 @@ def get_cosmetic_display_price(course, registration_price):
         return _('Free')
 
 
+class EnrollStaffView(View):
+    """
+    Display the enroll_staff Html page of about.html if user is not a global staff,
```

Comment:
We have described the actual purpose of the view. (e.g. it enrolls global staff in the to course).